testing/ncids-css-testing/stories/uswds-native/usa-icon-list/src/index.scss

```diff
@@ -1,3 +1,17 @@
 @use "styles/ncids";
 @forward "usa-icon-list";
 @forward "uswds-utilities";
+
+// NCIDS removes these basic colors and as such
+// we need toa dd here for use in twig templates
+// for storybook
+
+.text-red {
+	color: #e52207; //red-vivid-50
```

Comment:
you should still be able to use the `color` mixin with token values instead of hard coding